src/Intacct/Xml/Response/ErrorMessage.php

```diff
@@ -51,7 +51,7 @@ public function __construct(\SimpleXMLElement $errorMessage = null)
             $pieces = [];
             foreach ($error->children() as $piece) {
                 //strip out any tags in error messages
-                $piece = htmlspecialchars_decode($this->cleanse($piece), ENT_QUOTES);
+                $piece = htmlspecialchars_decode($this->cleanse($piece), ENT_QUOTES | ENT_HTML5);
```

Comment:
Can we verify `$piece` is a string? since we are now enforcing the type in `$this->cleanse()`?

Reply:
$piece in here is either a SimpleXMLElement (which implements `__toString()`), or `null`.  This means either can be casted to a string, which I went ahead and did (and added a strict_types declaration to the file).